policy.php

```diff
@@ -0,0 +1,37 @@
+<?php
+	require_once("template.php");
+	require_once("config.php");
+	Template::render_header("Privacy Policy");
+
+	echo "<h1>" . _("Imprint & Privacy Policy") . "</h2>";
+	echo "<h2>" . _("Contact") . "</h2>";
+	echo "##name## <br>";
+	echo "##email## <br>";
+	echo '<h2>' . _("Privacy Policy") . '</h2>';
+	echo '<h3>' . _("General") . "</h3>";
+	echo _("General
```

Comment:
This general should probably go away?

Reply:
It depens, But I will rewrite the policy to include some stuff @SigmaScorpii proposed.